From 2198cb5d508055f495af29866045d51c1098d5cc Mon Sep 17 00:00:00 2001
From: Ovidiu Sas <osas@voipembedded.com>
Date: Thu, 12 Jun 2014 12:46:39 -0400
Subject: [PATCH] core: fix crash related to core sockets_list  command  -
 reported by Nuno Reis <nreis@wavecom.pt> (cherry picked from commit
 1c715b1a2a66a4a15d1a8def33ef369212712bb1)

---
 core_cmd.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core_cmd.c b/core_cmd.c
index 304c49d..52150e9 100644
--- a/core_cmd.c
+++ b/core_cmd.c
@@ -936,7 +936,7 @@ static void core_sockets_list(rpc_t* rpc, void* c)
 						"proto", get_proto_name(proto),
 						"address", si->name.s);
 				if (!si->flags & SI_IS_IP)
-					rpc->struct_add(ha, "ss",
+					rpc->struct_add(ha, "s",
 						"ipaddress", si->address_str.s);
 				rpc->struct_add(ha, "sss",
 						"proto", si->port_no_str.s,
-- 
2.0.0

